src/test/java/mate/academy/dao/impl/RoleDaoImplTest.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.dao.impl;
+
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractTest {
+    private RoleDaoImpl roleDao;
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+    }
+
+    @Test
+    void save_Ok() {
+        Role role = new Role();
+        role.setRoleName(Role.RoleName.USER);
+        Assertions.assertNotNull(roleDao.save(role));
```

Comment:
All DAO save methods are better to test with checking record in DB to ensure that entity is actually saved, not only response